tests/test_util.py

```diff
@@ -6,17 +6,89 @@
 from canvasapi import Canvas
 from canvasapi.course import CourseNickname
 from canvasapi.user import User
-from canvasapi.util import combine_kwargs, obj_or_id
+from canvasapi.util import combine_kwargs, is_multivalued, obj_or_id
+from itertools import chain
+from six import integer_types, iterkeys, itervalues, iteritems
 from tests import settings
 from tests.util import register_uris
 
+try:
+    # in Python 2.x, iterator-returning zip function is "itertools.izip"
+    from itertools import izip as zip
```

Comment:
`six` provides this functionality as `six.moves.zip`. https://pythonhosted.org/six/
in py2 it's `itertools.izip()` and in py3 it's just `zip`